localstack/services/lambda_/invocation/lambda_models.py

```diff
@@ -599,6 +599,25 @@ class Function:
     def latest(self) -> FunctionVersion:
         return self.versions["$LATEST"]
 
+    # HACK to model a volatile variable that should be ignored for persistence
```

Comment:
I am also not 100% satisfied with this hack, but I think for now it is the best solution. Let's hope we can find a way to get by without this hack in the future :)

Reply:
yeah, pretty ugly hack 😢 
We currently don't have a Python object such as the VersionManager at function level. Alternative hacks:
* Use a dict to track them in the lambda service, but passing it down into the event managers is gonna be awkward
* Add it to the version manager of a specific version (e.g., v1, $LATEST is annoying because then it needs to migrate)